draft-twi-reference-architecture.md

```diff
@@ -49,72 +48,104 @@ informative:
+
+This document illustrates a gap analysis that is the output of the Confidential Computing Consortium identifying areas in the IETF WIMSE WG work where the current WIMSE architecture should be extended to accommodate workloads running in Confidential Computing environments. This document outlines high-level requirements for the these extensions and describes a series of use cases.
 
 --- middle
 
 # Introduction
-Until recently, there were few scenarios demanding data-in-use protections. This is starting to change. Regulatory bodies worldwide are increasingly requiring data-in-use protection and privacy-enhancing technologies. Outside of regulatory requirements, companies are exploring multiparty computation, machine learning training & inferencing, addressing the actual and perceived risks of computing in the public clouds, insider threats, and other reasons for protecting data-in-use. Correspondingly, there is an increased push to harmonize management and governance of human and non-human identities. Enterprises interested in strong assurances around the security of their deployed workloads, for regulatory, contractual and peace of mind reasons, will soon face large and challenging tasks of upgrading their existing IT systems to meet these requirements.
-Current ways of issuing and managing workload identities, as well as those required for effective protection of data-in-use, suffer from multiple architectural shortcomings; chief among them:
+Until recently, there were few scenarios requiring data-in-use protections. This is starting to change. Regulatory bodies worldwide are increasingly requiring data-in-use protection and privacy-enhancing technologies. Outside of regulatory requirements, companies are exploringi:
+
+* Multiparty computation
+* Machine learning training & inferencing
+* Addressing the actual and perceived risks of computing in the public clouds
+* Entrusting confidential data to SaaS providers
+* Insider threats, and other reasons for protecting data-in-use
+
+Correspondingly, there is an increased push to harmonize management and governance of human and non-human identities. Entities interested in strong assurances around the security of their deployed workloads, for regulatory, contractual or peace of mind reasons, are starting to face large and challenging tasks of upgrading their existing computing system infrastructure to meet these requirements.
+Current ways of issuing and managing workload identities, as well as those required for effective protection of data-in-use, are subject to multiple architectural challenges; chief among them:
 
 1. Lack of workload isolation against the hardware and the operating system owners/administrators, as well as peer workload instances
 2. Lack of strong binding between a workload credential and the workload instance to which that credential had been issued
 3. Inability to associate a credential with a set of decisions leading up to its issuance
```

Comment:
We might want to add "lack of certainty about the composition of the workload" which captures both weak attestation and lack of provenance information